### PR TITLE
[Vanilla Datepicker js] fixed DatepickerOptions prev arrow missing

### PR DESCRIPTION
**Pull request by camilosanchezdev**
Fixed DatepickerOptions previous arrow missing

### CODE REVIEW

**Review by MysteryBlokHed — COMMENTED** [review]
If you update the tests (vanillajs-datepicker-tests.ts) to show the new change, the PR can be merged without a maintainer's approval. For example:

```typescript
new Datepicker(container, { prevArrow: '<' });
```

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by nihaopaul** [comment]
Ready to merge

**Pull request merged**
